inst/qml/Ancova.qml

```diff
@@ -241,6 +241,49 @@ Form
 		}
 	}
 	
+	Section
```

Comment:
I tried this but it does not seem to work. Since today afternoon I cannot test the ANOVA module anymore, I dont know what happend, all other modules work. So I also cannot see my QML changes. Would leave it for now as it currently works.